sky/engine/bindings/jni/dart_jni.cc

```diff
@@ -85,6 +106,13 @@ DART_NATIVE_CALLBACK_STATIC(JniClass, FromName);
 
 FOR_EACH_BINDING(DART_NATIVE_CALLBACK)
 
+ScopedJavaGlobalRef<jobject> DartJni::class_loader_;
+ScopedJavaGlobalRef<jclass> DartJni::class_clazz_;
+jmethodID DartJni::class_loader_load_class_method_id_;
+jmethodID DartJni::class_get_name_method_id_;
+Dart_PersistentHandle DartJni::jni_object_type_;
+Dart_PersistentHandle DartJni::jni_float_type_;
```

Comment:
Shouldn't these handles be per isolate? I realize we don't support multiple isolates but it would be better if we put this into an object whose lifetime is tied to that of the isolate.